code/api/package.json

```diff
@@ -23,7 +24,10 @@
     "@babel/node": "7.8.7",
     "@babel/plugin-transform-runtime": "7.9.0",
     "@babel/preset-env": "7.9.5",
-    "nodemon": "2.0.3"
+    "jest": "^26.4.2",
+    "nodemon": "2.0.3",
+    "sequelize-cli": "^5.5.1",
+    "supertest": "^4.0.2"
```

Comment:
Packages we need for  BE testing

Reply:
Resource: https://sequelize.org/master/manual/migrations.html

Sequelize is a promise-based Node.js ORM for Postgres, MySQL, MariaDB, SQLite and Microsoft SQL Server. It features solid transaction support, relations, eager and lazy loading, read replication and more.